src/components/home/tests/home.test.tsx

```diff
@@ -1,8 +1,31 @@
-import { render } from "@testing-library/react";
-import "@testing-library/jest-dom/extend-expect";
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+  waitFor,
+  act,
+} from "@testing-library/react";
+// @see https://storybook.js.org/addons/@storybook/react-testing
+import { composeStories } from "@storybook/testing-react";
+// import all stories from the stories file
+import * as stories from "../stories/home.stories";
 
-import Home from "../home";
+// Every component that is returned maps 1:1 with the stories, but they already contain all decorators from story level, meta level and global level.
+const { HomeStory } = composeStories(stories);
+
+describe('components/home', () => {
+  test("renders home with testing-react", () => {
+    const { queryAllByText } = render(<HomeStory />);
+    expect(queryAllByText("Vulcan Next", { exact: false })).not.toBeNull();
+    expect(
+      queryAllByText("Vulcan Next", { exact: false }).length
+    ).toBeGreaterThan(0);
+  });
+});
 
+/* Old way - without testing-react
+import Home from "../home";
 describe("components/home", () => {
   test("renders home with React Testing", () => {
     const { queryAllByText } = render(<Home />);
```

Comment:
so you can remove those old comments, the only difference is that you render from a Story